NineChronicles.Headless/GraphTypes/ActionQueryFields/RapidCombination.cs

```diff
@@ -18,20 +19,20 @@ private void RegisterRapidCombination()
                     Name = "avatarAddress",
                     Description = "Avatar address to execute rapid combination"
                 },
-                new QueryArgument<NonNullGraphType<IntGraphType>>
+                new QueryArgument<NonNullGraphType<ListGraphType<IntGraphType>>>
                 {
-                    Name = "slotIndex",
-                    Description = "Slot index to execute rapid"
+                    Name = "slotIndexList",
```

Comment:
Also, since these changes are breaking-changes, you should check if there is anywhere that uses this query.